job/common/src/main/java/alluxio/job/plan/meta/PlanInfo.java

```diff
@@ -9,12 +9,13 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.job.meta;
+package alluxio.job.plan.meta;
```

Comment:
Does this need to be in the `meta` package, could we place it in `plan` directly?

Reply:
PlanInfo has a wire representation and a meta representation. The existence of both probably makes it such that it's better to have a meta folder.